packages/xo-web/src/common/render-xo-item.js

```diff
@@ -192,19 +192,24 @@ export const Sr = decorate([
       container: getContainer(state, props),
     })
   }),
-  ({ sr, container, link, newTab, spaceLeft }) => {
+  ({ sr, container, link, newTab, spaceLeft, self }) => {
     if (sr === undefined) {
       return UNKNOWN_ITEM
     }
 
     return (
       <LinkWrapper link={link} newTab={newTab} to={`/srs/${sr.id}`}>
         <Icon icon='sr' /> {sr.name_label}
-        {container !== undefined && (
-          <span className='text-muted'> - {container.name_label}</span>
+        {!self && spaceLeft && isSrWritable(sr) && (
```

Comment:
If self-users have the right to see the SR, why we don't display the SR's size for them

Reply:
We're supposed to hide as much information as possible about the underneath architecture to Self users. So everything that's not necessary should be hidden.

I don't think a Self user needs to know about the space left on the SR but we'll ask @olivierlambert about this :)

(Our spec even says that, ideally, a Self user shouldn't see the SR's name)

Reply:
That's correct. Self should give little info, think like AWS cloud: when you create a VM on Amazon cloud, you have no idea on where it's stored as a user, nor the physical space left on the storage.